### DISCUSSION TITLE
Permission issues on Ubuntu (possibly related to previous Netatalk?)

### DISCUSSION THREAD

**Discussion by cliffbeall** [Netatalk/netatalk#1835] [Q&A]
This is on a Dell Precision T5610. It is on its 3rd OS, previously had run Bio-Linux 8 (Ubuntu 14.04) and then Ubuntu MATE 18.04. It has two external RAID drives that have ext4 partitions. In the first two Linux configurations the drives were shared with Macs on a LAN using Netatalk. For various reasons, we no longer need sharing of this data.

I recently did a fresh install of Ubuntu MATE 24.04. As I had done in the previous transition, I created the same account name and id number as in the previous two systems. However I now get permission denied for the external drives, for commands including cd or ls. If I use sudo the commands work and I can see the files and directories are owned by the user and id I expect. I can create new directories and chown them (using sudo) and still get permission denied for the owning account.

Does Netatalk modify the disks in some way that would affect permissions globally? If so is it is easily reversible? Should I just install Netatalk on the new Ubuntu, even though I don't need to share the files?

Thanks so much for any help.

**Comment by rdmark** [comment]
Were you using ACLs with Netatalk previously? Try looking at how the file systems are mounted in fstab.

Cf. https://help.ubuntu.com/community/FilePermissionsACLs

**Reply by cliffbeall** [reply]
Thanksfor the reply. I looked over that page you referenced but it doesn't seem like that is what is going on. The fstab line was:

`/dev/disk/by-label/GLLab_QNAP /mnt/GLLab_QNAP auto nosuid,nodev,nofail,x-gvfs-show 0 `

I did notice that ACL may be automatically included in newer Ubuntu but getfacl on one of the affected directories only shows the normal linux permissions, at least if I understand it right.

`getfacl /mnt/GLLab_QNAP/GLLab_new_share/zShared`
`getfacl: Removing leading '/' from absolute path names`
`# file: mnt/GLLab_QNAP/GLLab_new_share/zShared`
`# owner: leyslab`
`# group: leyslab`
`user::rwx`
`group::rwx`
`other::r-x`

I have reverted back to Ubuntu MATE 18.04 for now and that is where I did the above. When I get some time I may boot back into 24.04 and see if anything is different there.
